conf/schemas/annotations.schema.js

```diff
@@ -0,0 +1,84 @@
+db.runCommand({
+  collMod: "annotations",
+  validator: {
+    $and: [
+      {
+        $or: [{ _user: { $type: "objectId" } }, { _user: { $exists: false } }],
+      },
+      {
+        _content: { $type: "object", $exists: true },
+      },
+      {
+        "_content.contentType": { $type: "string", $exists: true },
+      },
+      {
+        "_content._id": { $type: "string", $exists: true },
+      },
+      {
+        $or: [{ _task: { $type: "objectId" } }, { _task: { $exists: false } }],
```

Comment:
By default the fields are optional I think but if we want to define a type it is necessary to guard against the other case because the type comparison fails, if the field is non-existent.
For required fields you coud probably dont need the `$exists: true` but it seems to be best-practice.

Reply:
It this is best-practice, let's keep it this way.